src/components/cax-companyRole.tsx

```diff
@@ -85,6 +85,17 @@ export const CompanyRoleCax = () => {
     setAgreementChecked(updatedMap)
   }
 
+  const isNextBtnDisabled = () => {
+    if (companyRoleChecked && agreementChecked) {
+      if (
+        isObjectEmptyOrFalsy(companyRoleChecked) ||
+        isObjectEmptyOrFalsy(agreementChecked)
+      ) {
+        return true
+      }
+    }
+  }
```

Comment:
this method returns either `true` or `undefined` - would prefer to return always a boolean `true` or `false`. With a small change in `isObjectEmptyOrFalsy` I think you could write that more compact as:

```
const isNextBtnDisabled = () =>
  isObjectEmptyOrFalsy(companyRoleChecked) ||
  isObjectEmptyOrFalsy(agreementChecked)
```

Reply:
This makes more sense now.